### DISCUSSION TITLE
GEMPY - saving pyvista object

### DISCUSSION THREAD

**Discussion by skwskwskwskw** [gempy-project/gempy#586] [Q&A]
**What is your question?**
How to save the pyvista item with or without the interactive interface - where we're allowed to drag around? Scratching my head through tutorial.
![image](https://user-images.githubusercontent.com/7208012/84181289-461af700-aabb-11ea-862d-1eb0f73f020e.png)

**Comment by javoha** [comment]
Hi @similang ,
I figure a screenshot is not sufficient for your purposes? Maybe the Pyvista experts can help (@AlexanderJuestel and @Leguark).

**Comment by AlexanderJuestel** [comment]
Hello,
yes, it is fairly simple to do. In your `gp.plot_3d`, just add `notebook = True` and it will plot the figure in your notebook. However, the interactivity will be lost then.
`gpv = gp.plot_3d(geo_model, image=False, show_topography=True, plotter_type='basic', notebook=True)`
